docs/examples/configuration/test_example_7.py

```diff
@@ -0,0 +1,22 @@
+from dataclasses import dataclass
+from typing import Optional
+from uuid import UUID
+
+from polyfactory.factories.dataclass_factory import DataclassFactory
+
+
+@dataclass
+class Person:
+    id: UUID
+    name: Optional[str]
+
+
+class PersonFactory(DataclassFactory[Person]):
+    __model__ = Person
+    __allow_none_optionals__ = True
+    __random_seed__ = 0
+
+
+def test_optional_field() -> None:
```

Comment:
I don't think this might be the best example. If `__allow_none_optionals__` is set to `True`, then it does not guarantee that `None` is set for that fields. It just means that `None` is a possible value that is set for that fields. I think it might be better to document this properly and use an example with `__allow_none_optionals__` set to `False`.

Reply:
This makes sense. I've update to use an example that sets this to `False` and doesn't rely on setting the seed